assemble/bin/accumulo-cluster

```diff
@@ -652,6 +653,50 @@ function control_services() {
 
 }
 
+function prune(){
+  trap 'rm -f "$SERVICE_JSON"' EXIT
+  SERVICE_JSON=$(mktemp --tmpdir "accumulo-services-XXXXXXXX.json") || exit 1
+  "$accumulo_cmd" admin serviceStatus --json > $SERVICE_JSON 2>/dev/null 
```

Comment:
If using `serviceStatus` doesn't work out, another idea may be to create cluster.yaml file from a running system, then sort and diff them, to determine what things need to be stopped and started.

Reply:
Its working well so far. Manually tested the following cases of extra servers w/ the shell code and it find the extra stuff.

 * Group does not exists in cluster.yaml
 * Host does not exists in cluster.yaml
 * Count is lower than in cluster.yaml

Trying to figure out the best way to generalize the code for all server type as next step then will look into making it call zoo zap.

Reply:
The cluster.yaml format changed in 4.0, you probably know that already, but figured I would bring it up.